src/visualization/utils/colorMappingUtils.ts

```diff
@@ -159,7 +159,14 @@ export const generateSeriesToColorHex = (
   const cgMap = {...columnGroupMap}
   cgMap.mappings.forEach((graphLine, colorIndex) => {
     const id = getSeriesId(graphLine, columnGroupMap.columnKeys)
-    const colors = properties.colors ?? DEFAULT_LINE_COLORS
+    let colors
```

Comment:
This was coming in as an empty array `[]` and we only dealt with the cases when the `properties.colors` was either `null` or `undefined`. We needed to add the handler to deal with when it's `[]`.